### PR TITLE
O/noun 209 spike decrease homepage load times

### PR DESCRIPTION
**Pull request by olegakbarov**
- adds SSR to improve page load times
- fixes minor inconsistencies

### CODE REVIEW

File: services/collectionsService.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    const data = await zdk.collections({
+      where: { collectionAddresses: collectionAddresses },
+      sort: { sortDirection: SortDirection.Asc, sortKey: CollectionSortKey.None },
+    })
+
+    const collections = data.collections
+
+    return {
+      props: {
+        fallback: collections?.nodes,
```
Comment (bjfresh) [inline services/collectionsService.ts:17]: I feel like fallback isn't descriptive enough. Something like initialCollections would do a better job of clarifying what data is being passed.

File: services/collectionService.ts [hunk 2]
```diff
@@ -55,13 +55,15 @@ export async function collectionService({ params }: CollectionParamsProps) {
...
 
-    const seo = await buildCollectionSEO(name, symbol)
+    const seo = buildCollectionSEO(name, symbol)
 
     return {
       props: {
-        contractAddress: tokenAddress,
-        collection,
-        seo,
+        fallback: {
```
Comment (bjfresh) [inline services/collectionService.ts:62]: Maybe initialCollection rather than fallback?

**Review by bjfresh — APPROVED** [review]
@oleg approved, just small naming nits

File: services/collectionService.ts (hunk 2 above)
Comment (olegakbarov) [reply services/collectionService.ts:62]: Next use this naming in their docs and I like that it shorter then any `initialData` notation

**Pull request merged**